### ISSUE TITLE
Deprecating `trash` broke Ubuntu/Debian

### ISSUE THREAD

**Issue by nobodysu** [roddhjav/apparmor.d#71]
https://github.com/roddhjav/apparmor.d/commit/688a62e9bc7f4cbfc3197d7dab00dbb56dafbbe5
```
ubuntu2204# dpkg -S /etc/apparmor.d/abstractions/trash
dpkg-query: no path found matching pattern /etc/apparmor.d/abstractions/trash
```
```
debian11# dpkg -S /etc/apparmor.d/abstractions/trash
dpkg-query: no path found matching pattern /etc/apparmor.d/abstractions/trash
```
`AppArmor parser error for /etc/apparmor.d/nautilus in /etc/apparmor.d/nautilus at line 18: Could not open 'abstractions/trash'`

Should we move `trash` back with configure on Ubuntu/Debian?

**Comment by roddhjav** [comment]
I am aware of it. As the commit message says, the trash abstraction has been upstreamed. So I removed it. However, as usual only Archlinux is up to date. Therefore it broke Ubuntu and Debian install, [see the Gitlab pipeline](https://gitlab.com/roddhjav/apparmor.d/-/pipelines/630482877). I will fix it later today.

**Comment by Jeroen0494** [comment]
Is the following warning during building on Ubuntu 22.04 to be expected?
```
Ignoring request to rename file '/etc/apparmor.d/abstractions/trash' owned by diverting package 'apparmor.d'
*** OMINOUS WARNING ***: /etc/apparmor.d/abstractions/trash is not linked to either trash.apparmor.d or trash.apparmor.d-orig
```

**Comment by roddhjav** [comment]
Yes, my bad. I removed the file and put it back, so dpkg is kind of lost. Remove it manually and then install the package again and it should work.

**Comment by roddhjav** [comment]
Should actually be fixed by 0667c43

**Comment by Jeroen0494** [comment]
Thanks, I managed to replace the file manually by forcing an overwrite.
